src/include/txn.h

```diff
@@ -261,11 +273,13 @@ struct __wt_txn {
      *	ids < snap_min are visible,
      *	everything else is visible unless it is in the snapshot.
      */
-    uint64_t snap_min, snap_max;
+    uint64_t snap_max, snap_min;
     uint64_t *snapshot;
     uint32_t snapshot_count;
     uint32_t txn_logsync; /* Log sync configuration */
 
+    /* Save the snapshot backup. */
+    WT_TXN_SNAP_BACKUP snap_backup;
```

Comment:
better this be a pointer so we can control the memory usage.

Reply:
Fixed in 949c7e969da8a8592f25161144061faadef335cd